LICENSE.md

```diff
@@ -0,0 +1,4 @@
+Copyright 2017-2017 Analytical Graphics, Inc.
```

Comment:
This should include the Apache license, just copy what Cesium has up to (but not including `END OF TERMS AND CONDITIONS`): https://github.com/AnalyticalGraphicsInc/cesium/blob/master/LICENSE.md